namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-tier-dev/resources/rds.tf

```diff
@@ -0,0 +1,63 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ */
+
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+module "hmpps_tier_dev_rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.12"
+
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  team_name              = "manage-a-sentence"
+  business-unit          = "HMPPS"
+  application            = "manage-a-sentence"
+  is-production          = "false"
+  namespace              = var.namespace
+  environment-name       = "development"
+  infrastructure-support = "hmpps@digital.justice.gov.uk"
```

Comment:
You can get most of these from your variables (e.g. `var.team_name`) rather than hard-coding them here.